requirements/test.txt

```diff
@@ -0,0 +1,4 @@
+Django==2.0.5
```

Comment:
Dup

Reply:
@webknjaz without `Django==2.0.5` in `test.txt` Travis will be red.
I deleted `Django==2.0.5` from `base.txt`
Right? Or need another way?

Reply:
@webknjaz I think now all good. What do you say?